build.gradle

```
@@ -341,14 +365,11 @@ repositories { RepositoryHandler handler ->
     exclusiveRepo(handler, 'https://maven.parchmentmc.org/', 'org.parchmentmc.data')
 }
 
-tasks.named('test') {
-    //Disable builtin test task as we use and build uses testJunit so there is no point in having it also attempt to run an empty test task
-    enabled = false
+tasks.named('test').configure {
+    useJUnitPlatform()
```

Please add `org.gradle.api.tasks.testing.Test` as an import and then change this to:

```suggestion
tasks.named('test', Test) {
    useJUnitPlatform()
```

So that it can resolve the type of the task being configured for purposes of checking the method exists.